test/SpecHelper.hs

```diff
@@ -160,7 +160,7 @@ analyzeTable dbConn tableName =
 
 loadFixture :: Text -> FilePath -> IO()
 loadFixture dbConn name =
-  void $ readProcess "psql" ["--set", "ON_ERROR_STOP=1", toS dbConn, "-a", "-f", "test/fixtures/" ++ name ++ ".sql"] []
+  void $ readProcess "psql" ["--set", "ON_ERROR_STOP=1", toS dbConn, "-q", "-f", "test/fixtures/" ++ name ++ ".sql"] []
```

Comment:
The `hGetContents: invalid argument (invalid byte sequence)` is something that also happened on non-docker environments so it's good we don't have that anymore.